packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -175,12 +194,9 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     ];
     _imagePickerController.videoQuality = UIImagePickerControllerQualityTypeHigh;
 
-    self.result = result;
-    _arguments = call.arguments;
-
-    int imageSource = [[_arguments objectForKey:@"source"] intValue];
-    if ([[_arguments objectForKey:@"maxDuration"] isKindOfClass:[NSNumber class]]) {
-      NSTimeInterval max = [[_arguments objectForKey:@"maxDuration"] doubleValue];
+    int imageSource = [[call.arguments objectForKey:@"source"] intValue];
+    if ([[call.arguments objectForKey:@"maxDuration"] isKindOfClass:[NSNumber class]]) {
+      NSTimeInterval max = [[call.arguments objectForKey:@"maxDuration"] doubleValue];
```

Comment:
Since you're touching these lines anyway, please replace `objectForKey:` with the modern `[]` syntax. We're opportunistically updating old style as we change code.

Reply:
Done.